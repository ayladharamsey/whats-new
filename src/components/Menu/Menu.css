nav {
    height: 81vh;
    width: 20vw;
    display: flex;
    flex-direction: column;
    background-color: #B7E6D6;
}