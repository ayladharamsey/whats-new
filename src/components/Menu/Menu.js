import React from 'react';
import local from '../../data/local';
import health from '../../data/health';
import entertainment from '../../data/entertainment';
import science from '../../data/science';
import technology from '../../data/technology';
import './Menu.css'

const Menu = ({selectCategory}) => {
    return (
        <nav>
            <h3 onClick={event => selectCategory(technology)} >Tech</h3>
            <h3 onClick={event => selectCategory(local)}>Local</h3>
            <h3 onClick={event => selectCategory(entertainment)}>Entertainment</h3>
            <h3 onClick={event => selectCategory(science)}>Science</h3>
            <h3 onClick={event => selectCategory(health)}>Health</h3>
        </nav>
    )
}

export default Menu;