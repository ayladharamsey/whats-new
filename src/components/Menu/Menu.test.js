import Menu from './Menu'
import React from 'react'
import { shallow } from 'enzyme';

describe ('Menu', () => {
    let wrapper, category; 

    beforeEach(() => {
        category='potatos'
        wrapper = shallow(<Menu selectCategory={category} />)
    });

    it('should be able to access the selectCategory function ', () => {
        expect(wrapper).toMatchSnapshot();
    });

    it.skip('should be able to call selectCategory on click', () => {
        

    });



})