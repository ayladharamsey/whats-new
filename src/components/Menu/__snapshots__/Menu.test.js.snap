// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should be able to access the selectCategory function  1`] = `
<nav>
  <h1>
     What's New 
  </h1>
  <h3
    onClick={[Function]}
  >
    Local
  </h3>
  <h3
    onClick={[Function]}
  >
    Tech
  </h3>
  <h3
    onClick={[Function]}
  >
    Entertainment
  </h3>
  <h3
    onClick={[Function]}
  >
    Science
  </h3>
  <h3
    onClick={[Function]}
  >
    Health
  </h3>
</nav>
`;
